tests/User_selenium_unittest.py

```diff
@@ -0,0 +1,126 @@
+        driver.find_element_by_id("dropdownMenu1").click()
+        # "Click" the add prodcut button
+        driver.find_element_by_link_text("New User").click()
+        # Fill in the Necessary User Details
+        # username, first name, last name, email, and permissions
+        # Don't forget to clear before inserting
+        # username
+        driver.find_element_by_id("id_username").clear()
+        driver.find_element_by_id("id_username").send_keys("propersahm")
+        # First Name
+        driver.find_element_by_id("id_first_name").clear()
+        driver.find_element_by_id("id_first_name").send_keys("Proper")
+        # Last Name
+        driver.find_element_by_id("id_last_name").clear()
+        driver.find_element_by_id("id_last_name").send_keys("Samuel")
+        # Email Address
+        driver.find_element_by_id("id_email").clear()
+        driver.find_element_by_id("id_email").send_keys("propersam@example.com")
+        # Select user with super user permissions
+        driver.find_element_by_name("is_superuser").click()  # Clicking will mark the checkbox
+        # "Click" the submit button to complete the transaction
+        driver.find_element_by_css_selector("input.btn.btn-primary").click()
+        # Query the site to determine if the user has been created
+        productTxt = driver.find_element_by_tag_name("BODY").text
+        # Assert ot the query to dtermine status of failure
+        self.assertTrue(re.search(r'User added successfully, you may edit if necessary.', productTxt) or
+            re.search(r'A user with that username already exists.', productTxt))
+
+    def test_user_edit_permissions(self):
+        # Login to the site. Password will have to be modified
+        # to match an admin password in your own container
+        driver = self.login_page()
+        # Navigate to User Management page
+        driver.get(self.base_url + "user")
+        # Select A user to edit
+        driver.find_element_by_link_text("Edit").click()
+        # Unselect Super Admin Permission and only select Staff Permission
+        driver.find_element_by_name("is_superuser").click()
+        driver.find_element_by_name("is_staff").click()
+        # "Click" the submit button to complete the transaction
+        driver.find_element_by_css_selector("input.btn.btn-primary").click()
+        # Query the site to determine if the User permission has been changed
+        productTxt = driver.find_element_by_tag_name("BODY").text
+        # Assert ot the query to dtermine status of failure
+        self.assertTrue(re.search(r'User saved successfully.', productTxt))
+
+    def test_user_delete(self):
+        # Login to the site. Password will have to be modified
+        # to match an admin password in your own container
+        driver = self.login_page()
+        # Navigate to the product page
+        driver.get(self.base_url + "user")
+        # Select A user to edit
+        driver.find_element_by_link_text("Edit").click()
```

Comment:
This edits the admin user instead of the newly created one. it is not an option to delete the admin user.

Reply:
This is fixed. The newly created user is now filtered out before the edit button is clicked and then delete button.